buildpacks/jvm-function-invoker/Cargo.toml

```diff
@@ -2,23 +2,18 @@
 name = "jvm-function-invoker-buildpack"
 version = "0.0.0"
 edition = "2018"
-license = "MIT"
-
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-anyhow = "1"
-libcnb = "0.1.3"
-reqwest = { version = "0.11", features = ["blocking"] }
-serde = { version = "1.0", features = ["derive"] }
-sha2 = "0.9"
-termcolor = "1.1"
-toml = "0.5"
-
-openssl-sys = "*"
+libcnb = "0.2.0-prerelease1"
+serde = "1.0.126"
+toml = "0.5.8"
+sha2 = "0.9.5"
+thiserror = "1.0.25"
+reqwest = { version = "0.11.3", features = ["blocking"] }
+openssl = { version = "0.10", features = ["vendored"] }
+termcolor = "1.1.2"
```

Comment:
I think a few of these dependencies are now unused, since they are now dependencies of libherokubuildpack instead?

Reply:
Yes! Thanks, I'll do a cleanup of those.